Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -201,7 +201,8 @@ public abstract partial class BaseMeshSync : MonoBehaviour, IObservable<MeshSync
 
         #region Properties
         
-        private int currentSessionId = -1;
+        private int  currentSessionId  = -1;
+        private bool ignoreNewSessions = false;
```

Comment:
why do we need to ignore new sessions?

Reply:
We need a way to suppress the new session dialog when running meshsync from script.